config/__init__.py

```diff
@@ -0,0 +1,34 @@
+# flake8: noqa
+import os
+
+FLASK_ENV = os.getenv("FLASK_ENV")
+
+if not FLASK_ENV:
+    raise KeyError("FLASK_ENV does not exist in environ")
+
+match FLASK_ENV:
+    case "unit_test":
+        from config.envs.unit_test import UnitTestConfig as Config
+    case "development":
+        from config.envs.development import DevelopmentConfig as Config
+    # case "dev":
+    #     from config.envs.dev import (
+    #         DevConfig as Config,
+    #     )
+    # case "test":
+    #     from config.envs.test import (
+    #         TestConfig as Config,
+    #     )
+    # case "uat" | "production":
+    #     from config.envs.production import (
+    #         ProductionConfig as Config,
+    #     )
+    case _:
+        from config.envs.default import DefaultConfig as Config
+
+try:
+    Config.pretty_print()
```

Comment:
Is this just a user message? Do we need it, or can it be replaced with logging?

Reply:
It's from `fsd_utils` and prints the config into a table which shows where each value is defined when the app starts - seems useful to keep IMO

https://github.com/communitiesuk/funding-service-design-utils/blob/8ab0dd38130ea12039fb15d7f79224a57f51e665/fsd_utils/config/configclass.py#LL29C28-L29C28